app/views/users/deletion_info.html.erb

```diff
@@ -30,17 +30,32 @@ See doc/COPYRIGHT.rdoc for more details.
 <%= labelled_tabular_form_for :user, :url => user_path(@user), :html => { :method => :delete, :class => 'confirm_required' } do |form| %>
 
   <div class='wiki'>
-    <p>
-      <%= l("account.deletion_info.login_consequences.#{User.current == @user ? 'self' : 'other'}") %>
-    </p>
-
-    <p>
-      <%= l("account.deletion_info.data_consequences.#{User.current == @user ? 'self' : 'other'}") %>
-    </p>
-
-    <p>
-      <%= l("account.deletion_info.info.#{User.current == @user ? 'self' : 'other'}") %>
-    </p>
+    <section class="form--section">
+      <p>
+        <%= l("account.deletion_info.login_consequences.#{User.current == @user ? 'self' : 'other'}") %>
+      </p>
+      <p>
+        <%= l("account.deletion_info.data_consequences.#{User.current == @user ? 'self' : 'other'}") %>
+      </p>
+      <p>
+        <%= l("account.deletion_info.data_consequences.#{User.current == @user ? 'self' : 'other'}") %>
+      </p>
+      <p class="danger-zone--warning">
```

Comment:
I think we shouldn't change the DOM already here, since it should be done in https://github.com/opf/openproject/pull/3451